Library/Homebrew/test/rubocops/deprecate_disable/reason_spec.rb

```diff
@@ -324,4 +324,16 @@ class Foo < Formula
       RUBY
     end
   end
+
+  context "when auditing `deprecate!` and `disable!`" do
+    it "reports no offense if `reason` exists on `deprecate!` but is absent on `disable!`" do
```

Comment:
Was working on opposite (reason on `deprecate!`) and forgot to reverse message
```suggestion
    it "reports no offense if `reason` exists on `disable!` but is absent on `deprecate!`" do
```